src/PaymentElement.res

```diff
@@ -237,6 +244,25 @@ let make = (
         <React.Suspense fallback={loader()}>
           <BecsBankDebitLazy paymentType list />
         </React.Suspense>
+      | GooglePay =>
+        switch gPayToken {
+        | OtherTokenOptional(optToken) =>
+          switch googlePayThirdPartyToken {
+          | GooglePayThirdPartyTokenOptional(googlePayThirdPartyOptToken) =>
+            <GPayLazy
```

Comment:
Since this is a lazy loaded component, needs to be wrapped in React.Suspense. The other condition as well below.

Reply:
Brain fade moment ..Agreed but Lazy loading was being handled in the payment request element ...now that it has been moved out side..only this part will need the suspense not the other ..and payment request elemnet wont even need lazy loading since it is loaded from the begining itself  when the sdk loads

Reply:
Either one of the other will be rendered in the tree, so if this one renders, then lazy load will download gpay js bundle. So we can actually put suspense on both, only the one getting rendered will request for the bundle

Reply:
ok no harm in putting in both the places